test/backend/dyn_broadcast.in.cpp

```diff
@@ -73,3 +73,46 @@ NGRAPH_TEST(${BACKEND_NAME}, dyn_broadcast)
         ASSERT_TRUE(test::all_close_f(results, expected_results[i], MIN_FLOAT_TOLERANCE_BITS));
     }
 }
+
+NGRAPH_TEST(${BACKEND_NAME}, dyn_broadcast_default_axes)
+{
+    // Create a graph for
+    //   f(x,shape:i32) = Broadcast(x,Convert<i64>(shape)).
+    auto x = make_shared<op::Parameter>(element::f32, PartialShape{Dimension::dynamic()});
+    auto shape = make_shared<op::Parameter>(element::i32, Shape{3});
+    auto shape_i64 = make_shared<op::Convert>(shape, element::i64);
+
+    auto bc = make_shared<op::DynBroadcast>(x, shape_i64);
```

Comment:
Only one constructor is being tested.

Reply:
Constructor with 3 input is tested in a test above.